api_food/foodapi.py

```diff
@@ -0,0 +1,43 @@
+#access the USDA food database, for Take a Byte
+#
+
+import requests #pip3 install requests
+import json 
```

Comment:
nit: standard import "import json" should be placed before "import requests"pylint(wrong-import-order)
change order of imports please